src/Http/Controllers/VoyagerBaseController.php

```diff
@@ -891,6 +891,21 @@ public function relation(Request $request)
                     ];
                 }
 
+                // Sort results
```

Comment:
It works but I'd prefer to be implemented checking first the mandatory field like this:
```php
// Sort results
if (!empty($options->sort->field)) {
    if (!empty($options->sort->direction) && strtolower($options->sort->direction) == 'desc') {
        $relationshipOptions = $relationshipOptions->sortByDesc($options->sort->field);
    } else {
        $relationshipOptions = $relationshipOptions->sortBy($options->sort->field);
    }
}
```

Reply:
Why the adversion on using `empty`? :smile: 
I think it fits perfectly in this case.

WIth your conditions you are still not excluding when `$options->sort->field` is an empty string, it's not going to give an error but still it also not going to do anything inside that block of code.

Reply:
yeah, I'm inclined to agree with @MrCrayon.  `empty()` seems like the obvious choice here.

Reply:
Ok ok, another try.

Reply:
Ah! You are still missing the beauty of `empty` :cry: , you really don't need to check all the steps just use it as in my previous snippet
```php
if (!empty($options->sort->field)) {
```

Reply:
Err strange. That behaved different in previous PHP versions...well